core/src/main/java/com/netflix/conductor/core/execution/DeciderService.java

```diff
@@ -209,9 +210,16 @@ private DeciderOutcome decide(final WorkflowModel workflow, List<TaskModel> preS
                     outcome.tasksToBeUpdated.add(pendingTask);
                 } else {
                     pendingTask.setStatus(COMPLETED_WITH_ERRORS);
+                    workflow.setCompletedWithErrors(true);
                 }
             }
 
+            if (pendingTask.getTaskType().equals(TASK_TYPE_SUB_WORKFLOW)
```

Comment:
should we support this for other task types too ? besides subworkflow task types

Reply:
done